### ISSUE TITLE
Ignore html tag in sticky scroll

### ISSUE THREAD

**Issue by arjittw** [microsoft/vscode#163742]
Everyone knows when he is coding in html file he is working inside html tag.  
It's annoying to see a `<html>` tag always at the top

**Comment by LordMcSniff** [comment]
yes i agree, though i don't exactly see how you would differentiate between tags/ lines, 
but you could just add a line limit, for example taking the last five lines only or the first and the last 4.

This is especially a problem if you work with multiple panels so the screen space is already reduced, normally this doesn't matter, but if you work on a laptop without secondary monitor, half the screen is the tested web page and the top of the editor is html and bottom css/js, it takes up half of that already quarter of screen space.
